clang/tools/clang-linker-wrapper/LinkerWrapperOpts.td

```diff
@@ -184,6 +184,14 @@ def sycl_module_split_mode_EQ :
   Flags<[WrapperOnlyOption]>,
   HelpText<"Option that turns on split library with the given split mode">;
 
+def use_sycl_post_link_tool : Flag<["--", "-"], "use-sycl-post-link-tool">,
```

Comment:
I think we can move this to a TODO comment in the code instead of help comment. "Option will be removed when sycl-post-link tool becomes removed."

Reply:
done